netstandard/tools/HandlePackageFileConflicts.cs

```diff
@@ -64,23 +77,43 @@ private void EnsurePackageRanks()
         }
 
         /// <summary>
-        /// Examines items for conflicting targetpath and chooses the best item.
+        /// Examines items for conflicting keys and chooses the best item.
         /// </summary>
-        /// <param name="items"></param>
-        /// <returns></returns>
-        private ITaskItem[] HandleConflicts(ITaskItem[] items)
+        /// <param name="items">Items from which to remove conflicts</param>
+        /// <param name="getItemKey">Delegate to calculate key for an item</param>
+        /// <returns>Items without conflicts</returns>
+        private ITaskItem[] HandleConflicts(ITaskItem[] items, Func<ITaskItem, string> getItemKey)
         {
-            var winningItems = new Dictionary<string, ITaskItem>(StringComparer.OrdinalIgnoreCase);
+            Dictionary<string, ITaskItem> itemsByKeyUnused;
+
+            return HandleConflicts(items, getItemKey, out itemsByKeyUnused);
+        }
+
+        /// <summary>
+        /// Examines items for conflicting keys and chooses the best item.
+        /// </summary>
+        /// <param name="items">Items from which to remove conflicts</param>
+        /// <param name="getItemKey">Delegate to calculate key for an item</param>
+        /// <param name="itemsByKey">Dictionary of items by key without conflicts</param>
+        /// <returns>Items without conflicts</returns>
+        private ITaskItem[] HandleConflicts(ITaskItem[] items, Func<ITaskItem, string> getItemKey, out Dictionary<string, ITaskItem> itemsByKey)
```

Comment:
I think I would still call itemsByKey, winningItemsByKey so it is a little clearer.